src/flows/pipes/Notification/ExportTask.tsx

```diff
@@ -291,6 +291,9 @@ ${ENDPOINT_DEFINITIONS[data.endpoint]?.generateQuery(data.endpointData)}`
   }, [generateDeadmanTask, generateThresholdTask, data.thresholds])
 
   const handleTaskCreation = _ => {
+    event('Alert panel exported', {
```

Comment:
can you move this to the Export Modal Skipped event? we already have some things key'd off of that